controller/proxy-injector/webhook_test.go

```diff
@@ -154,158 +153,6 @@ func TestGetPatch(t *testing.T) {
 	})
 }
 
-func TestGetLabelForParent(t *testing.T) {
```

Comment:
Given we no longer have `GetLabelForParent()` I removed this test, but created #2560 to complete the webhook tests, that we can address in the 2.3 testing sprint.